src/lib/OpenEXR/ImfZstdCompressor.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
```

Comment:
Please run clang-format on this file with `-style=file`.

Reply:
Done